src/Integration.UnitTests/LocalServices/ErrorListInfoBarController.QualityProfileBackgroundProcessorTests.cs

```diff
@@ -43,7 +44,7 @@ public class ErrorListInfoBarController_QualityProfileBackgroundProcessorTests
         private ConfigurableHost host;
         private ConfigurableVsProjectSystemHelper projectSystem;
         private ConfigurableVsOutputWindowPane outputWindowPane;
-        private ConfigurableSolutionBindingSerializer bindingSerializer;
```

Comment:
Fix up of the old test code - now needs to supply the _IConfigurationProvider_ service instead of _ISolutionBoundSerializer_.